client/app/telecom/pack/xdsl/access/diagnostic/details/pack-xdsl-access-diagnostic-details.html

```diff
@@ -0,0 +1,32 @@
+<oui-modal
+    data-title="{{ :: 'xdsl_details_diagnostic_modal_title' | translate }}"
+    data-primary-label="{{ :: 'close' | translate }}"
+    data-primary-action="$ctrl.$state.go('^')"
+    data-secondary-label="{{ $ctrl.getRemainingChecksText() }}"
+    data-secondary-action="$ctrl.reload()"
+    data-secondary-disabled="$ctrl.diagnostic.remaining === 0"
+    data-on-dismiss="$ctrl.$state.go('^')"
+    data-loading="!$ctrl.diagnostic">
+
+    <p data-ng-if="!$ctrl.diagnostic" >
+        <span data-translate="xdsl_details_diagnostic_modal_loading"></span><br>
```

Comment:
To remove `br` tag you can :
```suggestion
        <span class="d-block" data-translate="xdsl_details_diagnostic_modal_loading"></span>
```

Reply:
Seems not resolved.

Reply:
I think this `<br>` is legitimate. I'm in a paragraph, and want to break a new line between two sentences. Looks pretty standard usage to me compared to a hacky css trick.

Reply:
Actually, `br` is not supposed to be used for this kind of purpose :)

https://www.w3.org/TR/html50/text-level-semantics.html#the-br-element

For example, screen-readers can't skim between `span` but it can between `p` !

To separate content into blocks, please use `p` :)

The `p` above should be a `div` then